.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+**Related Issue(s):**
+Please provide a title for this pull request.
+
+**Description:**
+Please provide a brief description of the changes you are proposing.
+
+**Checklist:**
+
+- \[ \] I have read and followed the [contributing guidelines](/CONTRIBUTING.md).
+- \[ \] I have included a README file for my project.
+- \[ \] I have updated the main README file where necessary.
+- \[ \] I have included a requirements.txt file.
```

Comment:
Not sure if requirements.txt is necessary here.

Reply:
Removed this line.